relay/pkg/header/integration_test.go

```diff
@@ -0,0 +1,292 @@
+package header
+
+import (
+	"context"
+	"math/big"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/keep-network/tbtc/relay/pkg/btc"
+	chainlocal "github.com/keep-network/tbtc/relay/pkg/chain/local"
+)
+
+func TestRelay_Integration(t *testing.T) {
+	ctx, cancelCtx := context.WithCancel(context.Background())
+	defer cancelCtx()
+
+	bc, err := btc.ConnectLocal()
+	if err != nil {
+		t.Fatal(err)
+	}
+	btcChain := bc.(*btc.LocalChain)
+
+	lc, err := chainlocal.Connect()
+	if err != nil {
+		t.Fatal(err)
+	}
+	localChain := lc.(*chainlocal.Chain)
+
+	headers := []*btc.Header{
+		{Hash: to32Bytes(0), Height: 0, PrevHash: to32Bytes(0), Raw: toBytes(0)},
```

Comment:
Can we clarify that we simulate the "zero" batch is already relayed on the host chain?